cheatsheets/gleam-for-python-users.md

```diff
@@ -650,19 +654,23 @@ case parse_int("123") {
 }
 ```
 
-In order to simplify this construct, we can use the `try` keyword that will:
+In order to simplify this construct, we can use the `use` expression with the
+`then` function from the `gleam/result` module. (This used to be the `try`
+keyword. See [the v0.27 release
+post]([https://gleam.run/news/v0.27-hello-panic-goodbye-try/#moving-beyond-try])
+for more information.)
```

Comment:
I don't think we need to mention `try` anymore, it's enough to mention the blessed path

Reply:
What `try`? :wink: